yarn.lock

```diff
@@ -0,0 +1,2013 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
looks like you ran yarn instead of npm, you'll want to make sure this file is not committed to git history